README.md

```diff
@@ -7,6 +7,7 @@ This package provides the following ROS2 type adaptations:
 | sensor_msgs/PointCloud2 | StampedPointCloud_PCL | pcl::PointCloud\<PointT\> |PCL.hpp|
 | sensor_msgs/Image | StampedImage_CV | cv::Mat |CV.hpp|
 | nav_msgs/OccupancyGrid | StampedOccupancyGrid_CV | cv::Mat |CV.hpp|
+| dem_msgs/DigitalElevationMap | StampedImage_CV | cv::Mat |CV.hpp|
```

Comment:
This should be StampedDigitalElevationImage_CV (?)

Reply:
Changed with StampedDigitalElevationMap_CV